packages/vxl/makefile

```diff
@@ -16,7 +16,7 @@ RANLIB:=$(shell which ranlib)
 
 export MAKE CMAKE CC CXX AR RANLIB
 
-checkout_cmd:=$(GIT) clone git@github.com:vxl/vxl.git
+checkout_cmd:=$(GIT) clone https://github.com/vxl/vxl.git
```

Comment:
I suggest doing something like `git checkout 3b74e3f8098f82be353c9cb5dbff46706a424b75` after cloning this repo.  This will give you a specific version of the code (in this case, the last release) so things don't break when they update their repo.

Reply:
definitely a good idea... just not sure which commit to use, they haven't updated their release branch in 5 years. maybe it'd be worth posting an issue to their account to see if they recommend a specific commit? otherwise, we could use a commit that is used to successfully build FS (currently haven't done this)